CONSOLE-4263: Add initial Content Security Policy for Console web application (#14156)
openshift/console: frontend/packages/console-dynamic-plugin-sdk/README.md

import { MonitoringIcon } from '@patternfly/react-icons';
```

## Content Security Policy

Console application uses [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)

> (CSP) in order to detect and mitigate certain types of attacks. By default, the list of allowed

Reviewer: ```suggestion
(CSP) to detect and mitigate certain types of attacks. By default, the list of allowed
```

[CSP sources](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/Sources)

> includes the document origin `'self'` as well as Console webpack dev server when running off-cluster.

Reviewer: ```suggestion
includes the document origin `'self'` and Console webpack dev server when running off-cluster.
```

> All dynamic plugin assets _should_ be loaded via `/api/plugins/<plugin-name>` Bridge endpoint which

Reviewer: ```suggestion
All dynamic plugin assets _should_ be loaded with the `/api/plugins/<plugin-name>` Bridge endpoint which
```
